Makefile

```diff
@@ -1,7 +1,7 @@
 export GO111MODULE=on
 export TF_LOG=DEBUG
 SRC=$(shell find . -name '*.go')
-SONARQUBE_IMAGE?=sonarqube:8.9.10-developer
+SONARQUBE_IMAGE?=sonarqube:9.5.0-developer
```

Comment:
Can we remove this. Another pr is just changing this to sonarqube:developer. https://github.com/jdamata/terraform-provider-sonarqube/pull/122/files

Reply:
Sure thing I can remove this!